### PR TITLE
GH-1960 Intro Hub Forgot Password

### PR DESCRIPTION
**Pull request by benstrumeyer**
* [*] Have you followed the guidelines in [CONTRIBUTING.md](../CONTRIBUTING.md)?
* [*] Have you checked to ensure there aren't other open [Pull Requests](../pulls) for the same update/change?
* [*] Have you added an explanation of what your changes do?
* [*] Does your submission pass tests?
* [*] Did you lint your code prior to submission?

- Add forgot password to intro hub

Ticket: https://cliqztix.atlassian.net/browse/GH-1960

### CODE REVIEW

File: src/utils/utils.js [hunk 1]
```diff
@@ -617,3 +617,58 @@ export function injectNotifications(tab_id, importExport = false) {
 export function isCliqzOffer(offer) {
 	return (offer && offer.origin === 'cliqz' && offer.type === 'offers' && offer.data);
 }
+
```
Comment (christophertino) [inline src/utils/utils.js:620]: @benstrumeyer Please move all of these functions back to app/panel/utils. Background utils should only be imported by modules in the background (src) context. This prevents adding unnecessary piggyback imports to the React app.  If you need to communicate between app/ and src/ code, you can either use message passing or add light functions to the import-free src/utils/common.js file.  In this case, you only need this functions in the React app so they should live under app directory.

Comment (benstrumeyer) [reply src/utils/utils.js:620]: That makes sense. I moved the functions back to where they were originally

File: app/Account/AccountActions.js [hunk 2]
```diff
@@ -166,6 +166,7 @@ export const logout = () => dispatch => (
 );
 
 export const resetPassword = email => dispatch => (
+	// console.log('resetting password...');
```
Comment (christophertino) [inline app/Account/AccountActions.js:169]: Remove

File: app/hub/Views/LogInView/LogInViewContainer.jsx [hunk 3]
```diff
@@ -13,7 +13,7 @@
 
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { validateEmail } from '../../../panel/utils/utils';
+import { validateEmail } from '../../../../src/utils/utils';
```
Comment (christophertino) [inline app/hub/Views/LogInView/LogInViewContainer.jsx:16]: Change back

File: app/panel/components/CreateAccount.jsx [hunk 4]
```diff
@@ -15,7 +15,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import ClassNames from 'classnames';
 import RSVP from 'rsvp';
-import { validateEmail, validateConfirmEmail, validatePassword } from '../utils/utils';
+import { validateEmail, validateConfirmEmail, validatePassword } from '../../../src/utils/utils';
```
Comment (christophertino) [inline app/panel/components/CreateAccount.jsx:18]: change back

File: app/panel/components/Login.jsx [hunk 5]
```diff
@@ -15,7 +15,7 @@ import React from 'react';
 import { Link } from 'react-router-dom';
 import ClassNames from 'classnames';
 import RSVP from 'rsvp';
-import { validateEmail } from '../utils/utils';
+import { validateEmail } from '../../../src/utils/utils';
```
Comment (christophertino) [inline app/panel/components/Login.jsx:18]: change back

File: app/shared-components/ForgotPassword/ForgotPassword.jsx [hunk 6]
```diff
@@ -12,11 +12,11 @@
  */
 
 import React from 'react';
-import { Link } from 'react-router-dom';
+import { withRouter, Link } from 'react-router-dom';
 import ClassNames from 'classnames';
-import { validateEmail } from '../utils/utils';
+import { validateEmail } from '../../../src/utils/utils';
```
Comment (christophertino) [inline app/shared-components/ForgotPassword/ForgotPassword.jsx:17]: change back

File: src/utils/utils.js (hunk 1 above)
Comment (christophertino) [reply src/utils/utils.js:620]: Still need to remove these functions.

Comment (benstrumeyer) [reply src/utils/utils.js]: I didn't save the file and all of these errors slipped through. Removed these guys, sorry about that.

File: app/Account/AccountActions.js (hunk 2 above)
Comment (benstrumeyer) [reply app/Account/AccountActions.js]: Removed!

File: app/hub/Views/LogInView/LogInViewContainer.jsx (hunk 3 above)
Comment (benstrumeyer) [reply app/hub/Views/LogInView/LogInViewContainer.jsx]: Updated import!

File: app/panel/components/CreateAccount.jsx (hunk 4 above)
Comment (benstrumeyer) [reply app/panel/components/CreateAccount.jsx]: Updated import!

File: app/panel/components/Login.jsx (hunk 5 above)
Comment (benstrumeyer) [reply app/panel/components/Login.jsx]: Updated import!

File: app/shared-components/ForgotPassword/ForgotPassword.jsx (hunk 6 above)
Comment (benstrumeyer) [reply app/shared-components/ForgotPassword/ForgotPassword.jsx]: Updated import!

File: app/shared-components/ForgotPassword/ForgotPassword.scss [hunk 7]
```diff
@@ -0,0 +1,169 @@
+/* FORGOT PASSWORD PANEL */
```
Comment (christophertino) [inline app/shared-components/ForgotPassword/ForgotPassword.scss:1]: @benstrumeyer add a document header to this file please

Comment (benstrumeyer) [reply app/shared-components/ForgotPassword/ForgotPassword.scss:14]: Added

**Pull request merged**